src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/internal/core/java/corrections/JavaDiagnosticsDefinition.java

```diff
@@ -49,6 +52,8 @@ public final class JavaDiagnosticsDefinition extends BaseKeyedLazyInstance<IJava
     public boolean isAdaptedForDiagnostics(JavaDiagnosticsContext context) {
         try {
             return getInstance().isAdaptedForDiagnostics(context);
+        } catch (IndexNotReadyException | ProcessCanceledException | CancellationException e) {
```

Comment:
This pattern should also be applied to [JavaCompletionDefinition](https://github.com/OpenLiberty/liberty-tools-intellij/blob/useLSP4IJ1201/src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/internal/core/java/completion/JavaCompletionDefinition.java) for consistency. Guessing I'm not seeing this in the current PR because you haven't synch'd up with the current branch yet which contains the updates for integrating LSP4Jakarta 0.2.0.